samples/drivers/entropy/README.txt

```diff
@@ -9,7 +9,7 @@ Sample for the entropy gathering driver
 Building and Running Project:
 
 This project writes an infinite series of random numbers to the
-console, 10 per second.
+console, 9 per second.
 
```

Comment:
Or is it simply as fast as possible, in which case I’d delete the last part saying “9 per second”

Reply:
It actually loops fetching 9 values, and waiting 1 second.
Seems this commit 3ff4d7409ff44bc7e088d0103cb1138a7f859379 reduced the number of random values to 9 (by using the 10th entry as an overflow marker), but did not change the description

Reply:
OK. +1 for docs then